src/server/images/models.js

```diff
@@ -23,4 +25,38 @@ export default class ImageManager {
       cb(new Error('database error'));
     });
   }
+  static getImageUrls(params, cb) {
+    // get all image Urls of item
+    const itemKey = params.itemKey;
+    const isThumbnail = params.isThumbnail || false;
```

Comment:
It can be called  getImageUrls({itmeKey, isThumbnail}, cb).
cb called with (err, urls) params.
now I'm using this function to get image urls of each item in getSavedPosts function.